test/fixtures.json

```diff
@@ -4135,7 +4135,7 @@
       }
     },
     {
-      "exception": "Expected property \"a.b.c\" of type ?Number, got undefined",
+      "exception": "Expected property \"a.b\" of type Object, got null",
```

Comment:
Also fixed error messages like this being deceptive,  `a.b.c` wasn't the issue, `a.b` was null, but was throwing because of `null['c']`.